src/main/java/views/AcesUp/AcesUp.flt.html

```diff
@@ -29,6 +29,11 @@
 
```

Comment:
I changed the names of the card because it still called clubs hearts spades and diamonds.